it.tests/src/main/java/com/venia/it/tests/CommerceTestBase.java

```diff
@@ -66,9 +46,21 @@
     public CQRule cqBaseRule = new CQRule(cqBaseClassRule.authorRule);
 
     protected static CQClient adminAuthor;
+    protected static boolean isCloudEnvironment;
 
     @BeforeClass
     public static void init() {
         adminAuthor = cqBaseClassRule.authorRule.getAdminClient(CommerceClient.class);
+        // check again if the environment may be a cloud environment (either skd on localhost, or an adobeaemcloud host)
+        isCloudEnvironment = adminAuthor.getUrl().getAuthority().contains("localhost")
```

Comment:
How does this condition work with a local or CircleCI AEM 6.5 instance?

On CircleCI for example, you can see that the 6.5 IT job skips the `testProductListPageWithSampleData65` and `testProductPageWithSampleData65` tests, which should not be the case.

Reply:
Right, I missed that. Then we cannot use this approach unfortunately.

Reply:
I removed those methods again and assert the canonical url with an `endsWith` instead.